src/frontend/e2e/tests/filters.spec.ts

```diff
@@ -0,0 +1,48 @@
+import {test,expect} from '@playwright/test'
+import {LoginPage} from '../page-objects/LoginPage'
+import {SamplesPage} from '../page-objects/SamplesPage'
+
+test.describe("Filter Test Suite",()=>{
+    let loginPage: LoginPage;
+    let samplePage: SamplesPage;
+
+    test.beforeEach(async ({ page })=>{
+        await page.setViewportSize({ width: 1600, height: 1200 });
+        loginPage = new LoginPage(page);
+        samplePage = new SamplesPage(page);
+        await loginPage.login('lbrambila@contractor.chanzuckerberg.com','Br@mb1la');
+    })
+
+    test("Genome Recovery filter by [complete] test",async ()=>{
```

Comment:
Test titles should follow the convention:
"Should filter genome recover by completion date"
There is no need for the word test